commands/bake.go

```diff
@@ -201,7 +197,21 @@ func runBake(dockerCli command.Cli, targets []string, in bakeOptions, cFlags com
 		return nil
 	}
 
-	resp, err := build.Build(ctx, nodes, bo, dockerutil.NewClient(dockerCli), confutil.ConfigDir(dockerCli), printer)
+	var lsg *localstate.StateGroup
+	if len(bo) > 1 {
```

Comment:
@tonistiigi I currently set the local state group if there is more than one target but maybe it would also be useful for a single target with bake so we have the definition and inputs attached. WDYT?